web/src/components/navbar/NavBarPopupMenu.tsx

```diff
@@ -166,26 +166,28 @@ export const NavBarPopupMenu = (props: Props): ReactElement => {
       id="menu-list-grow"
       onKeyDown={handleListKeyDown}
       data-testid={"nav-bar-popup-menu"}
+      className="padding-bottom-0"
     >
       <MenuItem
-        className={`display-flex space-between padding-y-1 ${props.hasCloseButton ? "" : "menu-item-title"}`}
-        disabled={!props.hasCloseButton}
+        className={`display-flex ${props.hasCloseButton ? "padding-y-205" : "padding-y-1"} menu-item-title`}
+        disabled={true}
       >
         <div className="text-bold">{guestOrUserName}</div>
-        {props.hasCloseButton && (
-          <button
-            className="right-nav-close fac fdr fjc"
-            aria-label="close menu"
-            data-testid={"close-button-nav-menu"}
-            onClick={(): void => {
-              analytics.event.mobile_menu_close_button.click.close_mobile_menu();
-              props.handleClose();
-            }}
-          >
-            <Icon className="font-sans-xl">close</Icon>
-          </button>
-        )}
       </MenuItem>
```

Comment:
So we were having so much trouble getting the title bar to not be hover-able / focus able while still having its child be clickable / in the tab order within MUI that I just said screw it and made it an absolute position component.